src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from 'react';
+import { Icon as IconifyIcon } from '@iconify/react';
+
+interface Props {
+
+  /**
+   * Name of icon in format provider:icon-name.
+   */
+  name: string;
+
+  /**
+   * Style object.
+   */
+  style?: React.CSSProperties;
+
+  /**
+   * Class name.
+   */
+  className?: string;
+
+  /**
+   * Size in pixels.
+   */
+  size?: number | typeof stringSizes[number];
+}
+
+const defaultProps = {
+  style: undefined,
+  className: undefined,
+  size: 'md' as const,
+};
+
+const stringSizes = ['md', 'sm', 'xs'] as const;
+
+/**
+ * TODO: Icon.
+ */
+export const Icon: FC<Props> = ({ name, style, className, size }) => {
```

Comment:
Let's wrap components with props into [memo](https://reactjs.org/docs/react-api.html#reactmemo) like this:
```
import React, { FC, memo } from 'react';
export const Icon: FC<Props> = memo(({...}) => {...})
```